lib/vyasa_web/router.ex

```diff
@@ -23,11 +23,16 @@ defmodule VyasaWeb.Router do
 
     get "/og/:filename", OgImageController, :show
 
+
     get "/", PageController, :home
-    live "/explore/", SourceLive.Index, :index
-    live "/explore/:source_title/", SourceLive.Show, :show
-    live "/explore/:source_title/:chap_no", SourceLive.Chapter.Index, :index
-    live "/explore/:source_title/:chap_no/:verse_no", SourceLive.Chapter.ShowVerse, :show
+
+    live_session :gen_anon_session,
+      on_mount: [{VyasaWeb.Session, :anon}] do
+      live "/explore/", SourceLive.Index, :index
+      live "/explore/:source_title/", SourceLive.Show, :show
+      live "/explore/:source_title/:chap_no", SourceLive.Chapter.Index, :index
+      live "/explore/:source_title/:chap_no/:verse_no", SourceLive.Chapter.ShowVerse, :show
+    end
     live "/gita/", GitaLive.Index, :index
```

Comment:
TODO: deprecate gita routes

Reply:
removed gita and text routes and associated source code 

<img width="1073" alt="image" src="https://github.com/ve1ld/vyasa/assets/38996397/5429b1b5-9d6a-4a5b-b6ab-59f6ce7e2ce1">